README.md

```diff
@@ -47,6 +47,8 @@ For larger quotas, please get in touch with Jens [directly](mailto:jens@censusma
 
 For performance reasons, and to avoid unnecessarily drawing down API quotas, **cancensus** caches data queries under the hood. By default, **cancensus** caches in R's temporary directory, but this cache is not persistent across sessions. In order to speed up performance, reduce quota usage, and reduce the need for unnecessary network calls, we recommend assigning a persistent local cache using `set_cancensus_cache_path(<local cache path>, install = TRUE)`, this enables more efficient loading and reuse of downloaded data. Users will be prompted with a suggestion to change their default cache location when making API calls if one has not been set yet. 
 
+Starting with version 0.5.1 **cancensus** will automatically check if locally cached data has been recalled by StatCan via the new data recall API implemented in [CensusMapper](https::/censusmapper.ca). StatCan occasionally detects and corrects errors in their census data release, and **cancensus** will download a list of recalled data at the start of each session and emit a warning if it detected locally cached data that has been recalled. Removal of the cached recalled data has to be done explicitly by the user via the `remove_recalled_chached_data()` function. If data was cached with **cancenus** versions prior to version 0.5.0 there is insufficient metadata to determine all instances of recalled cached data, but the package will check every time cached data is loaded and can identify recalled data at this point at the latest and issues a warning if recalled data is loaded.
```

Comment:
> ... will download a list of recalled data at the start of each session and emit a warning if it detected locally cached data that has been recalled.

What if users are offline and are working with saved or cached data? I am not sure a silent API check and download is something _all_ users would agree to.

Reply:
The download happens on the first invocation of get_census, will clarify that. If users are offline this fails gracefully, just emitting a warning message that recall data isn't available on the first run, and everything runs through normally.